docs/apv.md

```diff
@@ -278,10 +278,11 @@ De overtreding beschreven in lid 1 staat bekend als “spam”
 2. Het is in geen enkele situatie toegestaan om geld van mensen hun bank te stelen of om voertuigen te scammen, om spelers te overvallen, te ontvoeren of negatief te bejegenen bij het gemeentehuis of om facturen en/of andere items te stelen van spelers die bezig zijn met een burgerjob.
 3. Wanneer voertuigen gebruikt worden als betaal middel voor illegale dingen, denk aan loodsen, drugs, wapens, etc. vervalt de bescherming zoals beschreven in lid 2.
 4. Met criminele activiteiten zoals vermeld in lid 1 worden dingen met betrekking tot drugs, overvallen, witwassen etc bedoeld, en dus geen overtredingen.
-5. Bij een overtreding van de wet in lid 1 en/of lid 2 wordt de persoon gestraft volgens de 4e categorie en wordt het geld teruggegeven aan het slachtoffer. Wanneer de persoon het geld al heeft uitgegeven zal het resulteren in straf van de 8e categorie.
-6. Hou hiermee wel rekening dat het bij het vergrijp zoals beschreven in lid 1 niet de bedoeling is hiermee misbruik te maken van onwetendheid van mensen, dus vermijd het oplichten van nieuwe spelers.
-7. Het is voor criminelen toegestaan om mensen gedurende langere tijd te ondervragen en/of te martelen. Echter mag dit maximaal circa 2 uur duren. Er dient ook goede roleplay aan gekoppeld te worden. Dit is ter beoordeling aan een stafflid.
-8. Bij een overtreding van de regel in lid 6 wordt de persoon gestraft volgens de 1e categorie en zal het scenario onmiddellijk worden teruggedraaid.
+5. Wanneer iemand een kogelwerend vest, masker, holster o.i.d. draagt zal dit ook gezien mogen worden als criminele activiteit zoals vermeld in lid 1.
```

Comment:
denk dat dit beter is te melden als voorbeeld bij lid 4

Reply:
Aanpassing heb ik gedaan.